e2e-tests/README.md

```diff
@@ -24,11 +29,28 @@ npm run test
 npx playwright show-report
 ```
 
+- If you have any UI changes and want to update the snapshots, use the following commands:
```

Comment:
My snapshot tests failed and not sure if they were meant to be fail at the first time. If so, I think it is better we mention what you should expect and how to deal with it in the first time.

After run `npm run update-snapshots`, I can see my tests are passed 😄. 

![image](https://user-images.githubusercontent.com/94404633/175440362-bf00efa9-dac6-4d93-9487-d93b44fe0f3e.png)

Reply:
Lets add a small comment in `README.md` about the first time running test, you need create snapshot.

Reply:
This seems to be a recurring theme/issue. I also wasn't sure what to expect when running the tests for the first time. I think a dedicated section in the README would be good:

```
Running the tests for the first time
When running the tests for the first time, the tests may fail because the snapshots have not been created yet and the tests have nothing to compare to. Generate the initial snapshots by running `npm run update-snapshots` before running the tests. For more information about test snapshots, visit the official Playwright documentation. (links to https://playwright.dev/docs/test-snapshots) 
```